webapp/platform/components/src/generic_modal/generic_modal.tsx

```diff
@@ -105,7 +105,7 @@ export class GenericModal extends React.PureComponent<Props, State> {
                 this.onHide();
             }
             if (this.props.handleEnterKeyPress) {
-                this.props.handleEnterKeyPress();
+                this.props.handleEnterKeyPress(event);
```

Comment:
I was wondering if maybe place the `event.isComposing` validation here ¿? 🤔

Reply:
Yeah, that would make sense to me.

I'd also consider using an HTML form for this behaviour, but since that would affect so many places where this modal is used, that seems risky to introduce without a ton of extra testing